includes/editor-templates/templates.php

```diff
@@ -382,3 +382,40 @@ class="elementor-template-library-template-action elementor-button go-pro"
 		<img id="elementor-template-library-connect__background-image-<?php Utils::print_unescaped_internal_string( $image ); ?>" class="elementor-template-library-connect__background-image" src="<?php Utils::print_unescaped_internal_string( $base_images_url . $image ); ?>.png" draggable="false" loading="lazy" />
 	<?php endforeach; ?>
 </script>
+
+<script type="text/template" id="tmpl-elementor-template-library-connect-cloud">
```

Comment:
This template currently has the same look as current "connect view", but I created a custom one because:
- The layout might change for the cloud version anyway.
- Different URL for activation
- I wanted to keep the source dropdown as well so the user will be able to go back to his Site Library.
- We don't want to see the search and new folder icon.